openedx_tagging/core/tagging/rest_api/v1/permissions.py

```diff
@@ -58,3 +58,23 @@ def has_object_permission(self, request, view, obj):
         """
         obj = obj.taxonomy if isinstance(obj, Tag) else obj
         return rules.has_perm("oel_tagging.list_tag", request.user, obj)
+
+    def has_permission(self, request, view):
+        """
+        Returns True if the request user is allowed the given view on the Taxonomy model.
+
+        We override this method to avoid calling our view's get_queryset(), which performs database queries.
```

Comment:
Just out of curiosity, do you know why this is happening?

I didn't find a place where we are actually evaluating the query. Were the queries from the Taxonomy model or some other tables?